.grunt-config/sass.js

```diff
@@ -70,6 +70,13 @@ const sass = {
 				dest: 'assets/css/modules/ai',
 				ext: '.css',
 			},
+			{
```

Comment:
I believe the scss file should be included inside assets/dev/scss/frontend/_modules.scss.